en/mkdocs.yml

```diff
@@ -67,8 +67,6 @@ nav:
         - 'SAML': quick-starts/webapp-saml-sample.md
         - 'Query SAML Assertions': quick-starts/query-saml-assertions-sample.md
         - 'Use SAML Artifact Binding': quick-starts/use-artifact-binding-sample.md
-        - 'Configure Attributes': quick-starts/attribute-config-sample.md
-        - 'Integrate with a Custom Protocol': quick-starts/webapp-custom-sample.md
```

Comment:
why are these docs removed?

Reply:
They don't have the attachments. The docs have been written thinking that these files will be available shortly.

Reply:
in that case, shall we keep it commented instead of removing it so that we don't miss it out in the future?